web/src/features/hfiCalculator/RowManager.ts

```diff
@@ -0,0 +1,249 @@
+import { FireCentre, PlanningArea } from 'api/hfiCalcAPI'
+import { StationDaily } from 'api/hfiCalculatorAPI'
+import { groupBy, isNull, isUndefined, range } from 'lodash'
+import { getDailiesByStationCode, getDailiesForArea } from 'features/hfiCalculator/util'
+import { dailyTableColumnLabels } from 'features/hfiCalculator/components/DailyViewTable'
+import { weeklyTableColumnLabels } from 'features/hfiCalculator/components/WeeklyViewTable'
+import {
+  calculateDailyMeanIntensities,
+  calculateMaxMeanIntensityGroup,
+  calculateMeanIntensity,
+  calculateMeanIntensityGroupLevel
+} from 'features/hfiCalculator/components/meanIntensity'
+import { calculatePrepLevel } from 'features/hfiCalculator/components/prepLevel'
+import { isValidGrassCure } from 'features/hfiCalculator/validation'
+import { NUM_WEEK_DAYS } from 'features/hfiCalculator/constants'
+
+// the number of decimal places to round to
+const DECIMAL_PLACES = 1
```

Comment:
Already exists in https://github.com/bcgov/wps/blob/5ac0ad8e9ca1e8d777acdd5fe8f98ae2468c29a8/web/src/features/hfiCalculator/constants.ts#L1

Reply:
removed this declaration, importing from constants.ts instead